### ISSUE TITLE
404 error while creating API Client

### ISSUE THREAD

**Issue by Super976** [labd/terraform-provider-commercetools#250]
We are trying to create API Client using Terraform. but we are getting 404 error. 

terraform {
required_providers {
commercetools = {
source = "labd/commercetools"
version = "0.31.0"
}
}
}

provider "commercetools" {
client_id = "id"
client_secret = "secret"
project_key = "my-project"
scopes = "manage_project:my-project manage_api_clients:my-project view_api_clients:my-project"
api_url = "https://api.us-central1.gcp.commercetools.com/"
token_url = "https://auth.us-central1.gcp.commercetools.com/"
}

resource "commercetools_api_client" "my-api-client" {
name = "My API Client"
scope = ["manage_orders:my-project", "manage_payments:my-my-project"]
}

Error: Request returned status code 404

with commercetools_api_client.my-api-client,

on subscriptions.tf line 5, in resource "commercetools_api_client" "my-api-client":
5: resource "commercetools_api_client" "my-api-client" {

**Comment by Super976** [comment]
There is issue in version 0.31 and onwards with multiple scopes combination specifically manage_api_clients and view_api_clients to use with manage_project. whereas it works fine older version 0.28.

**Comment by onur9** [comment]
Hello @Super976 

Have you tested with the latest release? I couldn't reproduce this issue.

**Comment by tleguijt** [comment]
@Super976 did you by any chance switch commercetools projects?
I found that we can get a 404 if the project changes, and the API client resource is still in the state file. In that case it can't find it and errors out with a 404.

**Comment by Super976** [comment]
@tleguijt Yes, I changed the CT project. afterwards it gave me 404 error.  but strange thing is when I changed the provider version back to 0.28 (as we are using 0.28 version in all our environment) then i did not get 404 error any more. So What fixed this error. If old project reference was still state files then changing version should not fix it? 
If in future, we need to change the project then what is the recommended way?

Thanks for the input.
